src/cdk/drag-drop/parent-position-tracker.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import {ViewportRuler} from '@angular/cdk/scrolling';
-import {getMutableClientRect, adjustClientRect} from './client-rect';
+import {adjustClientRect, getMutableClientRect} from './client-rect';
```

Comment:
Note that reordering the imports will make it harder to land and will cause conflicts with existing PRs.

Reply:
> Note that reordering the imports will make it harder to land and will cause conflicts with existing PRs.

That's my IDE config configuration, it always sorts it alphabetically :)
Anyway, if it causes too much noise, I can disable this configuration and rollback it.

Reply:
It might be best to roll it back since it causes a lot of churn for existing PRs.